app/blog/2017-07-12-an-update-on-neon-animation.md

```diff
@@ -0,0 +1,70 @@
+Well, rest easy, because nothing is actually changing. By deprecated we mean that we will keep the element as-is. There won't be any major changes or any new features coming to it, so if it works for you, it should keep working for you.
+
+## Why is this happening?
+Animations provide continuity between your page’s states through movement. This continuity should help users easily identify objects on your page and relationships between them. For example, a page sliding animation triggered by a user changing tabs shows the spatial relationship of two pages and provides the user an idea of the interaction model of your site as they can now imagine the shape of the off-screen elements. An animation should typically take place when a user interacts or immediately after and thus represents a transition to a state.
+
+On the other hand, the element model represents a state of an object; an input can be enabled or disabled, a paper-button can be elevated or flat, it can be at position *x* and *y*, but it can't be *going to be at x' y'*. The element model can provide expectation and relation via its properties and values over time, but an element cannot *be* expectation or relation. This is where `neon-animation` went wrong: it tried to shove the entire animation space into the element model which is similar but fundamentally incompatible.
+
+This resulted in the implementation of animations using `neon-animation` being more-complicated than they need to be. Making performant animations is hard enough as it is. The `neon-animation` API requires you to use a proprietary JSON configuration object instead of the native animation interfaces (CSS keyframe animations, CSS transitions, and the web animations API) which already describe the states of an animation clearly.
+
+Animations are incredibly powerful, but thinking about animations through `neon-animations` limited the scope of what could be achieved.
+
+For these reasons, we're not planning on maintaining the package going forward.
+
+## What should I do?
+If you're creating a new element from scratch, we recommend using CSS keyframe animations, CSS transitions, or the Web Animations API.
+
+If you're using another element from the PolymerElements org that depends on `neon-animation`, like `iron-dropdown` or `paper-menu-button`, don't sweat it. We'll replace these animations.
+If you're using `neon-animation` directly, you should start planning to eventually move off it.
+
+## What about sharing animations?
+`neon-animation` gave the appearance of having a shared approach to animations, but in reality it provided a non-standard, ill-typed JSON structure that would then configure and fill in the parameters for the Web Animations API. This unfortunately locked you into our ill-defined animations API, which would then lock you into the technical debt of all the features of `neon-animations` that you were not using, and all we wanted was to apply a set of pre-defined parameters into the Web Animations API.
```

Comment:
Whole paragraph: recommend using present tense throughout. `neon-animation` still exists, so you can use the present tense ("gives the appearance").

`ill-typed` and `ill-defined` is maybe overdoing it. Maybe cut `ill-typed` (which is a little awkward) and just keep ill-defined?

2nd sentence has a complicated tense "locked you in... which then ... lock you in..." And changes from "you" to "we." Maybe simpler:

This unfortunately locks you into our ill-defined animations API, which also locks you into the technical debt of all the features of `neon-animations` that you don't need. When all you want is to apply a set of pre-defined parameters into the Web Animations API.

Although, having written that, I don't really know what "the technical debt of all the features ... you don't need." means. If we're not going to explain that, maybe just stick with the one explanation (configuration is non-standard). I think you could cut this clause without weakening your argument.

This unfortunately locks you into our ill-defined animations API, when all you want is to apply a set of pre-defined parameters into the Web Animations API.